client/payment-details/dispute-details/index.tsx

```diff
@@ -4,8 +4,17 @@
  * External dependencies
  */
 import React from 'react';
+/**
+ * Internal dependencies
+ */
+import { Dispute } from '../../types/disputes';
```

Comment:
Nit: add `type` keyword and import from "absolute" path to add clarity.
```suggestion
import type { Dispute } from 'wcpay/types/disputes';
```

Reply:
TIL !